### PR TITLE
fix: use checksummed DAI address

### PR DESCRIPTION
**Pull request by troggy**

### CODE REVIEW

File: dapp/src/config.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ var txDefaultOrig =
       'decimals': 18
     },
     {
-      'address': '0x6b175474e89094c44da98b954eedeac495271d0f',
+      'address': '0x6B175474E89094C44Da98b954EedeAC495271d0F',
```
Comment (MaxStalker) [inline dapp/src/config.js:46]: Is this one correct now? :)

File: dapp/src/partials.js [hunk 2]
```diff
@@ -476,7 +476,7 @@ angular.module('multiSigWeb').run(['$templateCache', function($templateCache) {
     "      Payout Bounty\n" +
     "    </button>\n" +
     "    <button type=\"button\" class=\"btn btn-primary btn-lg\" disabled-if-no-accounts\n" +
-    "      ng-click=\"withdrawToken(wallet.tokens['0x6b175474e89094c44da98b954eedeac495271d0f'])\">\n" +
+    "      ng-click=\"withdrawToken(wallet.tokens['0x6B175474E89094C44Da98b954EedeAC495271d0F'])\">\n" +
```
Comment (MaxStalker) [inline dapp/src/partials.js:479]: I was wondering, why we are not using address from config file? 🤔
Same applies to line 498 and changes inside `partials/wallet`

File: dapp/src/config.js (hunk 1 above)
Comment (troggy) [reply dapp/src/config.js:46]: Works for me :)

File: dapp/src/partials.js (hunk 2 above)
Comment (troggy) [reply dapp/src/partials.js:479]: Just because it was a quick hack :)

**Review by MaxStalker — APPROVED** [review]

**Pull request merged**